README.md

```diff
@@ -93,10 +93,9 @@ If a file is not properly formatted, the exit code will be non-zero.
 
 ```console
 foo@bar:~$ mdformat --help
-usage: mdformat [-h] [--check] [--version] [--number] [--wrap {keep,no,INTEGER}]
-                [--end-of-line {lf,crlf,keep}] [--exclude PATTERN]
-                [--extensions EXTENSION] [--codeformatters LANGUAGE]
-                [paths ...]
+usage: mdformat [-h] [--check] [--no-validate] [--version] [--number]
+                [--wrap {keep,no,INTEGER}] [--end-of-line {lf,crlf,keep}]
+                [--extensions EXTENSION] [--codeformatters LANGUAGE] [paths ...]
```

Comment:
This is now missing `[--exclude PATTERN]` (which only appears on Python 3.13+).

If you don't have Python 3.13 at hand, maybe just manually add it, and word wrap at around 80 wide max.